src/ai/backend/common/events.py

```diff
@@ -529,6 +529,15 @@ def __call__(
 ]
 
 
+class KernelLifecycleEventReason:
```

Comment:
The recommended way to group multiple values as a single category is to use `enum.Enum` (ref: https://github.com/lablup/backend.ai/blob/bede3c289c0a2b3620d07e061d986f52898654d9/src/ai/backend/common/types.py#L232). Please update this class to inherit `enum.Enum`. Don't forget to update type definition on code which accepts `KernelLifecycleEventReason` also!